src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -28,16 +28,16 @@ type UseEstimateTransactionGasProps = {
 }
 
 type TransactionGasEstimationResult = {
-  gasPrice: string
-  gasPriceFormatted: string
-  gasMaxPrioFee: string
-  gasMaxPrioFeeFormatted: string
+  gasPrice?: string
+  gasPriceFormatted?: string
+  gasMaxPrioFee?: string
+  gasMaxPrioFeeFormatted?: string
 }
 
 export const calculateTotalGasCost = (
   gasLimit: string,
   gasPrice: string,
-  gasMaxPrioFee: string,
+  gasMaxPrioFee: string | undefined,
```

Comment:
```suggestion
  gasMaxPrioFee?: string,
```

Reply:
I reverted the change since we always have a fallback for `gasMaxPrioFee` prior to coming into this function

Reply:
Perfect 👍